android/app-newm/src/main/java/io/newm/AppLaunchGhostActivity.kt

```diff
@@ -18,18 +28,33 @@ class AppLaunchGhostActivity : ComponentActivity() {
         }
 
         if (userSession.isLoggedIn()) {
-            launchHomeActivity()
+            lifecycleScope.launch {
+                try {
+                    val user =
+                        userDetailsUseCase.fetchLoggedInUserDetailsFlow().filterNotNull().first()
+                    featureFlagMager.setUserId(user.id)
+                } catch (e: Exception) {
+                    logger.error(
+                        tag,
+                        "Failed to identify feature flag because fetching user details failed.",
+                        e
+                    )
+                } finally {
+                    launchHomeActivity()
```

Comment:
Nice, so we will always attempt to fetch the FF, but if it fails we will use default values.